apps/console/src/features/roles/components/edit-role/edit-role-groups.tsx

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com).
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```

Comment:
Shall we keep the year as it is?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/4326/commits/317543c8be8c0461214e6e67425f492c835d6bd7